<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenArkClass</class>
 <widget class="QMainWindow" name="OpenArkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenArk</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabProcess">
       <attribute name="title">
        <string>Process</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabKernel">
       <attribute name="title">
        <string>内核</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidgetKernel">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabGdt">
           <attribute name="title">
            <string>Gdt</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabKernelHook">
       <attribute name="title">
        <string>内核钩子</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidgetKernelHook">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabSSDT">
           <attribute name="title">
            <string>SSDT</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tabShadowSSDT">
           <attribute name="title">
            <string>Shadow SSDT</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabObjDir">
       <attribute name="title">
        <string>ObjDirectory</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/openark.qrc"/>
 </resources>
 <connections/>
</ui>
